multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -162,6 +165,59 @@
 }
 VALID_METRICS = METRIC_MODE_MAP.keys()
 
+# Define valid val/eval metrics and the fallback metric for each problem type.
+EVALUATION_METRICS = {
+    BINARY: METRICS[BINARY].keys(),
+    MULTICLASS: METRICS[MULTICLASS].keys(),
+    REGRESSION: METRICS[REGRESSION].keys(),
+    NER: [OVERALL_F1, NER_TOKEN_F1],
+    NAMED_ENTITY_RECOGNITION: [OVERALL_F1, NER_TOKEN_F1],
+    FEATURE_EXTRACTION: [None],  # No metrics needed for feature extraction
+    ZERO_SHOT_IMAGE_CLASSIFICATION: [None],  # TODO: No eval metrics needed for zero shot?
+    OBJECT_DETECTION: DETECTION_METRICS,
+    OPEN_VOCABULARY_OBJECT_DETECTION: DETECTION_METRICS,
+    OCR: [None],  # OCR is a WIP feature
+    OCR_TEXT_DETECTION: [None],  # OCR is a WIP feature
+    OCR_TEXT_RECOGNITION: [None],  # OCR is a WIP feature
+    IMAGE_SIMILARITY: [None],  # use matching metrics
+    TEXT_SIMILARITY: [None],  # use matching metrics
+    IMAGE_TEXT_SIMILARITY: [None],  # use matching metrics
+    FEW_SHOT_CLASSIFICATION: METRICS[BINARY].keys() + METRICS[MULTICLASS].keys(),
+    SEMANTIC_SEGMENTATION: [BINARY_IOU],  # TODO: ADD METRICS FOR SEMANTIC_SEGMENTATION
+}
+EVALUATION_METRICS_FALLBACK = {problem_type: metrics[0] for problem_type, metrics in EVALUATION_METRICS.items()}
+EVALUATION_METRICS_FALLBACK[BINARY] = ROC_AUC
+EVALUATION_METRICS_FALLBACK[MULTICLASS] = ACCURACY
+EVALUATION_METRICS_FALLBACK[REGRESSION] = RMSE
+EVALUATION_METRICS_FALLBACK[FEW_SHOT_CLASSIFICATION] = ACCURACY
+VALIDATION_METRICS = {
```

Comment:
We can replace previous `VALID_METRICS` with `VALIDATION_METRICS`. No need to keep them both.